app/src/store.tsx

```diff
@@ -19,9 +20,14 @@ interface AddCredential {
   addCredentialPressed: boolean
 }
 
+interface DismissPersonCredentialOffer {
+  personCredentialOfferDissmissed: boolean
```

Comment:
This is nitpicky, but I believe this should be `personCredentialOfferDismissed` with one less S.

Reply:
good catch, resolved